### PR TITLE
Add workflow to create podvm binaries image

### PR DESCRIPTION
**Pull request by bpradipt**
Tested locally using 
``` 
act workflow_dispatch -j build -W .github/workflows/podvm_binaries.yaml  | tee out.log
```

### CODE REVIEW

**Comment by bpradipt** [comment]
This adds the missing workflow which is required to create the release artefacts for 0.5.0

**Review by stevenhorsman — APPROVED** [review]
LGTM. Thanks @bpradipt

File: .github/workflows/podvm_binaries.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        tags: |
+          quay.io/confidential-containers/podvm-binaries-${{ matrix.os }}-${{ matrix.arch }}:latest
+          quay.io/confidential-containers/podvm-binaries-${{ matrix.os }}-${{ matrix.arch }}:${{ github.sha }}
+        push: true
+        context: podvm
+        platforms: linux/amd64
+        file: |
+          podvm/${{ matrix.dockerfile }}
+        build-args: |
+          "CLOUD_PROVIDER=${{ matrix.provider }}"
```
Comment (yoheiueda) [inline .github/workflows/podvm_binaries.yaml:62]: This is just a question. Where is `matrix.provider` is defined, and where is `CLOUD_PROVIDER` used?

Comment (bpradipt) [reply .github/workflows/podvm_binaries.yaml:62]: @yoheiueda good catch. This is a left over and is not used during the gh-action execution.

**Review by yoheiueda — APPROVED** [review]
LGTM. Thank you very much!

**Pull request merged**